manifests/bases/kubernetes-nmstate-operator.clusterserviceversion.yaml

```diff
@@ -18,17 +18,17 @@ metadata:
     categories: OpenShift Optional
     description: |
       Kubernetes NMState is a declaritive means of configuring NetworkManager.
-    containerImage: quay.io/openshift/origin-kubernetes-nmstate-operator:4.13
+    containerImage: quay.io/openshift/origin-kubernetes-nmstate-operator:4.14
```

Comment:
I'm curious why these changes are showing up in the PR diff since they shouldn't be a change from what the previous tree looked like. Maybe just the ordering of the PR merges?